pylabnet/launchers/pulsemaster.py

```diff
@@ -0,0 +1,21 @@
+""" Launches the wavemeter monitor/control application """
+
+from pylabnet.launchers.launcher import Launcher
+from pylabnet.scripts.pulsemaster import pulsemaster
+
+
+def main():
+
+    launcher = Launcher(
+        script=[pulsemaster],
+        server_req=[None],
+        gui_req=[None],
+        params=[None],
+        config='pulsemaster',
+        script_server=False
```

Comment:
why is script_server False? do you want to create a custom server in `pulsemaster.launch()`? Otherwise this `script_server` is used to terminate the pulsemaster thread and clear it from the launcher properly.

Reply:
This was not on purpose, will set the flag to True.